Makefile

```diff
@@ -0,0 +1,9 @@
+PKG:=github.com/docker/docker-network
+PKG_PATH:=/go/src/$(PKG)
+RUN_CMD:=docker run --privileged --rm -v "$(shell pwd)":$(PKG_PATH) -w $(PKG_PATH) golang:1.4 go
+
+build:
```

Comment:
the problem with running a build locally is that we can't make any guarantees about the environment it'll be built on, hence the container.